t/15-assets.t

```diff
@@ -102,11 +102,9 @@ $jobA->set_prio(1);
 # register worker
 my $c = OpenQA::WebAPI::Controller::API::V1::Worker->new;
 my $w;
-eval { $w = $c->_register($schema, 'host', '1', $workercaps); };
-like($@, qr/Incompatible websocket API version/, 'Worker no version - incompatible version exception');
+throws_ok { $w = $c->_register($schema, 'host', '1', $workercaps); } qr/Incompatible websocket API version/, 'Worker no version - incompatible version exception';
```

Comment:
That should work without the inner ;

Reply:
And the same counts for all the other occurrences. I also suggest to omit the inner `;`.

Reply:
yea. I did take care of the other situation. Except one which the test failed once I removed it